tests/integration/cli/scripts_test.py

```diff
@@ -33,13 +33,16 @@ def tearDown(self):
     def test_basic(self):
         self.assertFileNotInContainer('appa', '/app/a/foo')
         self.run_command('scripts appa example')
+        self.wait_for_exec_docker(1)
```

Comment:
I think it makes sense to provide a reasonable default, something like 3 seconds. If a test is going longer than that it's probably a crappy test, hopefully seeing someone override the default timeout would tip off a reviewer that there's a problem.